ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -118,23 +190,30 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
           message: '^Email address is invalid'
         }
       },
-      'verifiedInstitutionalAffiliation.institutionalRoleEnum': {
+      'profile.verifiedInstitutionalAffiliation.institutionalRoleEnum': {
         presence: {
           allowEmpty: false,
           message: '^Institutional role cannot be blank',
         }
       },
     };
+    if (!isBlank(this.state.profile.verifiedInstitutionalAffiliation.institutionShortName) &&
+        !isBlank(this.state.profile.contactEmail)) {
+      validationCheck['checkEmailResponse'] = {
+        checkEmailResponse: {}
+      };
+    }
+
     if (this.state.profile.verifiedInstitutionalAffiliation.institutionalRoleEnum === InstitutionalRole.OTHER) {
-      validationCheck['verifiedInstitutionalAffiliation.institutionalRoleOtherText'] = {
+      validationCheck['profile.verifiedInstitutionalAffiliation.institutionalRoleOtherText'] = {
         presence: {
           allowEmpty: false,
           message: '^Institutional role text cannot be blank',
         }
       };
     }
 
-    return validate(this.state.profile, validationCheck);
+    return validate(this.state, validationCheck);
```

Comment:
why change this?

Reply:
This was a kind of subtle but interesting change – by providing the entire state object to validate.js, it allows us to run checks against any property of the component's state (which is the most flexible for future additions). In this case, I wanted to add some validation on the `checkEmailResponse` property, to ensure our form submission is appropriately blocked if the email-validation check hasn't completed.

Reply:
nice, thanks